fw/http_msg.c

```diff
@@ -868,6 +868,19 @@ __hdr_expand(TfwHttpMsg *hm, TfwStr *orig_hdr, const TfwStr *hdr, bool append)
 	return tfw_strcpy(append ? &it : orig_hdr, hdr);
 }
 
+static void
+__hdr_del_from_tbl(TfwHttpHdrTbl *ht, unsigned int hid)
+{
+	if (hid < TFW_HTTP_HDR_RAW) {
+		TFW_STR_INIT(&ht->tbl[hid]);
+	} else {
+		if (hid < ht->off - 1)
```

Comment:
```suggestion
		if (hid + 1 < ht->off)
```
Or is it guaranteed here that `ht->off` is never 0? `hid` is always lower than 2^32, so this is safer